src/decorators/sub-store.spec.ts

```diff
@@ -0,0 +1,243 @@
+import { NgZone, Component, Injectable } from '@angular/core';
```

Comment:
yeah, or if `bar$` is defined in `Foo` - but the decorator is on `Blerg` - would `@select` perform as expected.

Or at least be able to document that behaviour